purview/models/koji.py

```diff
@@ -32,6 +32,11 @@ class KojiBuild(PurviewStructuredNode):
     version = StringProperty()
 
 
+class ContainerKojiBuild(KojiBuild):
```

Comment:
How about `KojiContainerBuild`?

Reply:
Haha, I had this dilemma yesterday as well, but then I thought that since we have a model class called KojiBuild, naming it ContainerKojiBuild might just give a hint of connection between the two classes to someone who just sees the labels. I know it sounds stupid, but what do you think?

Reply:
Sure, either one works for me.